subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -67,7 +69,8 @@ uint8_t ll_big_create(uint8_t big_handle, uint8_t adv_handle, uint8_t num_bis,
 	 * the periodic advertising trains is already associated
 	 * with another BIG.
 	 */
-	if (!adv || !adv->lll.sync || adv->lll.sync->adv_iso) {
+	lll_adv_sync = adv->lll.sync;
+	if (!adv || !lll_adv_sync || lll_adv_sync->iso) {
```

Comment:
Either the `!adv` is unneeded, or else it need to come before the dereference.

Reply:
hmm... i had fixed it I thought, may be I messed up with merge conflicts when rebasing the draft PR... will fix it.